### PR TITLE
NTTA: implemented UX Review hints

### PR DESCRIPTION
**Pull request by thomas-kaltenbach**

### CODE REVIEW

File: file-formats/ntta/type/zif_aff_ntta_v1.intf.abap [hunk 1]
```diff
@@ -44,9 +44,9 @@ INTERFACE zif_aff_ntta_v1
       "! <p class="shorttext">General Information</p>
       "! General information
       general_information TYPE ty_general_information,
-      "! <p class="shorttext">Assigned Notetypes</p>
-      "! Notetypes assigned to the SAP Object Node Type
-      assigned_notetypes  TYPE ty_notetypes,
+      "! <p class="shorttext">Note Types</p>
+      "! Note Types assigned to the SAP Object Node Type
+      notetypes  TYPE ty_notetypes,
```
Comment (albertmink) [inline file-formats/ntta/type/zif_aff_ntta_v1.intf.abap:49]:
```suggestion
      note_types  TYPE ty_notetypes,
```
ABAP is snake_case and gets converted to camelCase in JSON

File: file-formats/ntta/type/zif_aff_ntta_v1.intf.abap [hunk 2]
```diff
@@ -17,21 +17,21 @@ INTERFACE zif_aff_ntta_v1
...
-    "! <p class="shorttext">Notetypes</p>
-    "! Notetype
-    ty_notetypes TYPE SORTED TABLE OF ty_notetype_structure WITH UNIQUE KEY notetype,
+    "! <p class="shorttext">Note Types</p>
+    "! Note Types
+    ty_notetypes TYPE SORTED TABLE OF ty_notetype_structure WITH UNIQUE KEY name,
 
-    "! <p class="shorttext">Notetype Assignments</p>
-    "! Notetype assignments
+    "! <p class="shorttext">Note Type Assignments</p>
```
Comment (albertmink) [inline file-formats/ntta/type/zif_aff_ntta_v1.intf.abap:33]: Same as for NTTY

**Review by albertmink — CHANGES_REQUESTED** [review]
Some small upper/minor case issues.

File: file-formats/ntta/type/zif_aff_ntta_v1.intf.abap (hunk 1 above)
Comment (thomas-kaltenbach) [reply file-formats/ntta/type/zif_aff_ntta_v1.intf.abap]: done

File: file-formats/ntta/type/zif_aff_ntta_v1.intf.abap (hunk 2 above)
Comment (thomas-kaltenbach) [reply file-formats/ntta/type/zif_aff_ntta_v1.intf.abap]: done

**Review by albertmink — APPROVED** [review]
Looks good to me.

File: file-formats/ntta/type/zif_aff_ntta_v1.intf.abap [hunk 3]
```diff
@@ -17,21 +17,21 @@ INTERFACE zif_aff_ntta_v1
       implementation_exit TYPE zif_aff_types_v1=>ty_object_name_30,
     END OF ty_general_information,
 
-    "! <p class="shorttext">Notetype</p>
-    "! Notetype
+    "! <p class="shorttext">Note Type</p>
+    "! Note Type
```
Comment (schneidermic0) [inline file-formats/ntta/type/zif_aff_ntta_v1.intf.abap:21]: Is "Note Type" planned to take up as SAPterm (written in upper case)? Currently I only find "node type" there.

Other object types like classes, tables, programs are usually written in lower case.

If "note type" is the right term, you should change the descriptions to 
```suggestion
    "! Note type
```

Same is valid for "Note type assignment"

**Review by schneidermic0 — COMMENTED** [review]
Just a short question wrt upper and lower case of the terms "node type" and "node type assignment"

File: file-formats/ntta/type/zif_aff_ntta_v1.intf.abap (hunk 3 above)
Comment (thomas-kaltenbach) [reply file-formats/ntta/type/zif_aff_ntta_v1.intf.abap]: I changed it to lower case. We have no plans to add it to SAPterm.

**Review by schneidermic0 — APPROVED** [review]
LGTM. Thanks 👍

**Pull request merged**